aibolit/patterns/method_siblings/method_siblings.py

```diff
@@ -19,26 +19,30 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-from typing import List
-from aibolit.types_decl import LineNumber
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
-
-import javalang
+from aibolit.ast_framework.ast_node import ASTNode
+from typing import List
 import re
 
 
 class MethodSiblings:
-    def __init__(self):
-        pass
-
-    def value(self, filename: str) -> List[LineNumber]:
+    '''
+    Count those methods, which
+    start with the same names
+    '''
+    def __compare_next_nodes(self, node: ASTNode, new_node: ASTNode) -> List[int]:
+        node_name = re.findall(r'[A-Z]?[a-z]+|[A-Z]+(?=[A-Z]|$)', node.name)
+        if node.node_index < new_node.node_index and len(node_name) > 1 and \
+           new_node.name.startswith(node_name[0]):
+            return [node.line, new_node.line]
+        return []
 
-        numbers = []
-        for _, node in build_ast(filename).filter(javalang.tree.MethodDeclaration):
-            composed = re.findall(r'[A-Z]?[a-z]+|[A-Z]+(?=[A-Z]|$)', node.name)
-            if len(composed) > 1:
-                for _, inner in build_ast(filename).filter(javalang.tree.MethodDeclaration):
-                    if node.name != inner.name:
-                        if inner.name.startswith(composed[0]):
-                            numbers.append(node.position.line)
-        return numbers
+    def value(self, filename: str) -> List[int]:
+        numbers: List[int] = []
+        method_nodes = AST.build_from_javalang(
```

Comment:
we better declare `ast` on separate line.

Reply:
done!